### ISSUE TITLE
The site needs styling and other updates 

### ISSUE THREAD

**Issue by Anmol-Prasad** [Naman-sharma00100/Introduction-webpage#21]
Hi I reviewed your site and it lacks styling and responsiveness. You can assign this issue to me  I can fix everything

**Comment by Naman-sharma00100** [comment]
Yes I was waiting for the Hacktoberfest to start. I will assign you once it has started so that it will count as your contribution.

**Comment by Naman-sharma00100** [comment]
Hey Now you can work on this issue and send me the PR. I will review it.

**Comment by Naman-sharma00100** [comment]
@Anmol-Prasad

**Comment by Anmol-Prasad** [comment]
Sure I am working on it

**Comment by 0xYashwanth** [comment]
Hey @Naman-sharma00100, I have a pretty decent knowledge of CSS, and I want to contribute to the project.
Could you please assign this to me?
Thanks for reading!
